### ISSUE TITLE
Sorting of zones in start cleaning pop-up in reverse alphabetical order

### ISSUE THREAD

**Issue by pidator** [rand256/valetudo#73]
The order of the saved zones in the "start to clean" pop-up is in reverse alphabetical order (Z to A). In the zones-Tab the zones are shown in correct alphabetical order (A to Z).  Is this a known behaviour or should the order in the pop-up be in alphabetical order too?

**Vacuum Model:** Gen1

**Valetudo Version:** 0.8.1

**Comment by rand256** [comment]
I have equal sorting order in both zones tab and zoned cleaning popup on home tab.

**Comment by pidator** [comment]
Hi, just checked the gui again:
I'm using the german language setting, but with default (english) it's the same behaviour. Even a reboot of the robot doesn't change the order. I've also deleted the vowel mutation, still the same!
I just realised the order in the popup isn't really reverse alphabetical, there are some entries in between... it just looked reverse at the first look. The order is the same when using a different browser (tested with Android Chrome and Android Opera).

This are the configured zones in zones tab:
![zonestab](https://user-images.githubusercontent.com/59822549/72254849-7ee69b00-3605-11ea-8cee-390d4afed9f3.jpg)

and this is the view when the pop up appears for selecting a zone:
![popup](https://user-images.githubusercontent.com/59822549/72254875-8e65e400-3605-11ea-85a9-d04337f0fbf0.jpg)
